vhost/src/vhost_user/gpu_backend_req.rs

```diff
@@ -1,13 +1,56 @@
-use crate::vhost_user::connection::Endpoint;
```

Comment:
Can we avoid those changes?

I mean following the https://rust-lang.github.io/rustfmt/?version=master&search=import#StdExternalCrate%5C%3A from the beginning

Reply:
It seems you are referencing old commit here, but IIUC you mean grouping related imports together?